### ISSUE TITLE
Modulregistrering går sönder vid minifiering

### ISSUE THREAD

**Issue by andersarpi** [Hanifb/marginal-calc#7]
Om man kör detta lib genom en standardmässig JS-minifierare (om man nu vill baka in det i något annat man gör) så går modulregistreringen sönder, då den är beroende av att namnen på funktionsargumenten som blir utlästa i `getArguments(func)` matchar de strängar man skickar in till  `marginalCalc.scriptLoader.addScript([...])`.

Se till exempel [här](https://docs.angularjs.org/tutorial/step_05#a-note-on-minification) för hur det skulle kunna lösas.

**Comment by Hanifb** [comment]
Ja, Angulars lösning verkar vara den smidigaste. Lägger in en commit ikväll.
